applications/models.py

```diff
@@ -281,6 +281,17 @@ class WinterStorageApplication(BaseApplication):
         verbose_name=_("trailer registration number"), max_length=64, blank=True
     )
 
+    def is_unmarked_ws_application(self) -> bool:
+        first_area = self.chosen_areas.first()
+
+        if (
+            self.chosen_areas.count() == 1
+            and first_area.number_of_unmarked_spaces
+            and first_area.number_of_unmarked_spaces > 0
+        ):
+            return True
+        return False
```

Comment:
```suggestion
        return (
            self.chosen_areas.count() == 1
            and first_area.number_of_unmarked_spaces
            and first_area.number_of_unmarked_spaces > 0
        )
```

Reply:
or maybe what Alex suggested but with `bool` = `return bool(...)`

Reply:
if you want it to be more explicit, but Alex' suggestion is more pythonic

Reply:
👍 done